### DISCUSSION TITLE
Unable to uninstall

### DISCUSSION THREAD

**Discussion by webbb82** [espanso/espanso#1508] [General]
Hey first let me say i really want to try this. I  spend a few hours trying to get things working yesterday but at the end just got the error sorry your system isnt compatable with this please install another version. so thats why i am trying to uninstall. 
BUT
I dont know what to do. I have tried the gui way of uninstalling, i tried going to snap/espanso and just del the folders and it wont let me, its acting like its running but typing espanso stop in terminal says its not running. I tried command line rmdir i triedso many things and it wont go away. I just want it gone so i can reinstall the other version or something. any ideas or help weould be amazing

**Comment by AucaCoyan** [comment]
Hi! Sorry I'm late to the thread, we usually respond in [discord](https://discord.gg/DFcCNDg7bB).
Did you succeeded to uninstall the previous version?
